integration-tests/src/local.rs

```diff
@@ -175,10 +182,23 @@ impl Node {
         };
 
         let eth = mpc_node::indexer_eth::EthArgs {
-            eth_account_sk: Some(config.cfg.eth.account_sk.clone()),
-            eth_rpc_ws_url: Some(config.cfg.eth.rpc_ws_url.clone()),
-            eth_rpc_http_url: Some(config.cfg.eth.rpc_http_url.clone()),
-            eth_contract_address: Some(config.cfg.eth.contract_address.clone()),
+            eth_account_sk: config.cfg.eth.clone().map(|eth| eth.account_sk.clone()),
+            eth_consensus_rpc_http_url: config
+                .cfg
+                .eth
+                .clone()
+                .map(|eth| eth.consensus_rpc_http_url.clone()),
+            eth_execution_rpc_http_url: config
+                .cfg
+                .eth
+                .clone()
+                .map(|eth| eth.execution_rpc_http_url.clone()),
+            eth_contract_address: config
+                .cfg
+                .eth
+                .clone()
+                .map(|eth| eth.contract_address.clone()),
+            eth_network: None,
         };
```

Comment:
seems like this is getting redone enough to warrant us doing a function for this conversion from the config object